packages/browser-wallet-api-helpers/README.md

```diff
@@ -74,6 +74,15 @@ if (accountAddress) {
 }
 ```
 
+### getSelectedChain
```

Comment:
can we make a task to make use of this API in the wCCD and other examples. Right now there is a hacky solution there.

Reply:
Sure, I'll put an issue in the backlog.